src/iface-modbus.c

```diff
@@ -111,11 +223,11 @@ int iface_modbus_read_data(int reg_addr, int bit_offset, knot_value_type *out)
 
 	switch (bit_offset) {
 	case TYPE_BOOL:
-		rc = connection_interface.read_bool(modbus_ctx, reg_addr,
+		rc = read_bool(modbus_ctx, reg_addr,
 						    &tmp.val_bool);
 		break;
 	case TYPE_BYTE:
-		rc = connection_interface.read_byte(modbus_ctx, reg_addr,
+		rc = read_byte(modbus_ctx, reg_addr,
 						    byte_tmp);
```

Comment:
Can you check if this parameter can be fitted in the previous line?
If it can't please allign the first column with the function's first parameter column